sqlx-postgres/src/types/geometry/line.rs

```diff
@@ -0,0 +1,174 @@
+use crate::decode::Decode;
+use crate::encode::{Encode, IsNull};
+use crate::error::BoxDynError;
+use crate::types::Type;
+use crate::{PgArgumentBuffer, PgHasArrayType, PgTypeInfo, PgValueFormat, PgValueRef, Postgres};
+use sqlx_core::bytes::Buf;
+use sqlx_core::Error;
+use std::str::FromStr;
+
+const ERROR: &str = "error decoding LINE";
+
+/// Postgres Geometric Line type
+///
+/// Storage size: 24 bytes
+/// Description: Infinite line
+/// Representation: {A, B, C}
+///
+/// See https://www.postgresql.org/docs/16/datatype-geometric.html#DATATYPE-LINE
+#[derive(Debug, Clone, PartialEq)]
+pub struct PgLine {
+    pub a: f64,
+    pub b: f64,
+    pub c: f64,
```

Comment:
nit:
I would prefer if you were to copy the documentation from postgres in the regads what this is.
Maybe even with an `# Example`, what `a`,`b`,`c` represents. I'd assume users would not figgure this out without reading the external docs.

Good documentation is a superpower 😉

Reply:
Have addressed by copying in some more of the Postgres docs.